build/Dockerfile

```diff
@@ -38,14 +38,16 @@ RUN GOOS=windows go build ./cmd/containerd-shim-runhcs-v1
 WORKDIR /build/windows-machine-config-operator/kubelet/
 COPY kubelet/ .
 ENV KUBE_BUILD_PLATFORMS windows/amd64
-RUN make WHAT=cmd/kubelet
 RUN make WHAT=vendor/k8s.io/component-base/logs/kube-log-runner
+WORKDIR /build/windows-machine-config-operator/
+COPY Makefile Makefile
+RUN make kubelet
 
 # Build kube-proxy
 WORKDIR /build/windows-machine-config-operator/kube-proxy/
 COPY kube-proxy/ .
-ENV KUBE_BUILD_PLATFORMS windows/amd64
-RUN make WHAT=cmd/kube-proxy
+WORKDIR /build/windows-machine-config-operator/
+RUN make kube-proxy
```

Comment:
this can be consolidated

```
WORKDIR /build/windows-machine-config-operator/
COPY Makefile Makefile
COPY kubelet/ kubelet/
RUN make kubelet
COPY kube-proxy/ kube-proxy/
RUN make kube-proxy
```

Reply:
I addressed this slightly differently given we have to build the log-runner before.